examples/fairmotion_interface.py

```diff
@@ -435,23 +452,19 @@ def define_preview_points(joint_names: List[str]) -> List[mn.Vector3]:
 
         # TODO: This function is not working. It is supposed to produce a gradient
         #       from RED to YELLOW to GREEN but it is producing a black solely
-        colors = [
-            mn.Color3(255, 0, 0),
-            mn.Color3(255, 255, 0),
-            mn.Color3(0, 255, 0),
-        ]
+        colors = [mn.Color3(255, 0, 0), mn.Color3(255, 255, 0), mn.Color3(0, 255, 0)]
```

Comment:
```suggestion
        colors = [mn.Color3(1.0, 0, 0), mn.Color3(1.0, 1.0, 0), mn.Color3(0, 1.0, 0)]
```

Reply:
This is weird, I am pretty sure I pushed this line to be using the bindings `Color3.red()` etc.